### ISSUE TITLE
Design Pipeline flows into stages

### ISSUE THREAD

**Issue by staylorTT** [tenstorrent/tt-mlir#255]

**Comment by staylorTT** [comment]
@vmilosevic Is this still a relevant issue? Looks like I had minimal details when I opened this so i am unsure if we still need this.

**Comment by vmilosevic** [comment]
No, we can close this one. We have already separated build and test stages.
